cogs/sudo.py

```diff
@@ -374,6 +268,62 @@ async def gpuinfo(self, ctx: Context) -> None:
 
         await ctx.send(embed=embed)
 
+    @sudo.command()
+    @check(is_owner)
+    async def cmduser(self, ctx: Context, user: t.Union[Member, User], *, command: str) -> None:
+        """Run a command as another user."""
+        msg = copy.copy(ctx.message)
+        msg.author = user
+        msg.content = ctx.prefix + command
+
+        new_ctx = await self.bot.get_context(msg)
+        await self.bot.invoke(new_ctx)
+
+    @sudo.command()
+    @check(is_owner)
+    async def install(self, ctx: Context, package: str) -> None:
+        "Install a package"
+        async with ctx.typing():
+            os.system(f"pip install {package}")
+        await ctx.message.add_reaction("✅")
+
+        await ctx.send(f"Installed {package}!")
+
+    @sudo.command()
+    @check(is_owner)
+    async def update(self, ctx: Context, package: str) -> None:
+        "Update a package"
+        async with ctx.typing():
+            os.system(f"pip install --upgraded {package}")
+        await ctx.message.add_reaction("✅")
+
+        await ctx.send(f"Updated {package}!")
```

Comment:
Similarly to review above, this doesn't make sense, we'll have access to the server this runs on, no need for a command like this.
```suggestion
```